### ISSUE TITLE
Hiding fields not working after upgrade

### ISSUE THREAD

**Issue by acim** [kubeapps/kubeapps#1784]
### Description:

After upgrading kubeapps from Helm chart version 3.4.3 to 3.7.0, fields hiding doesn't work even for kubeapps itself. All fields are shown and if I enable/disable a fields which control hiding, nothing happens, everything is always shown. I could prove the same behaviour in IE Edge and Firefox.

### Steps to reproduce the issue:

1. Install kubeapps Helm chart 3.7.0

### Describe the results you received:

Fields that should be hidden are actually shown.

### Describe the results you expected:

Hidden fields not shown.

### Additional information you deem important (e.g. issue happens only occasionally):

I didn't see any significant errors in the JavaScript console.

### Version of Helm, Kubeapps and Kubernetes:

- Output of `helm version`:

```
version.BuildInfo{Version:"v3.2.2", GitCommit:"a6ea66349ae3015618da4f547677a14b9ecc09b3", GitTreeState:"clean", GoVersion:"go1.13.12"}
```

- Output of `helm list <kubeapps-release-name>`:

```
NAME    	NAMESPACE	REVISION	UPDATED                                	STATUS  	CHART         	APP VERSION
kubeapps	kubeapps 	2       	2020-06-08 13:39:04.998863196 +0000 UTC	deployed	kubeapps-3.7.0	v1.10.1
```

- Output of `kubectl version`:

```
Client Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.3", GitCommit:"2e7996e3e2712684bc73f0dec0200d64eec7fe40", GitTreeState:"clean", BuildDate:"2020-05-20T12:52:00Z", GoVersion:"go1.13.9", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"16", GitVersion:"v1.16.9", GitCommit:"a17149e1a189050796ced469dbd78d380f2ed5ef", GitTreeState:"clean", BuildDate:"2020-04-16T23:15:50Z", GoVersion:"go1.13.9", Compiler:"gc", Platform:"linux/amd64"}
```

**Comment by acim** [comment]
![Capture](https://user-images.githubusercontent.com/2791202/84040440-ecbfa480-a9a2-11ea-8d5f-d2708b323db4.PNG)

**Comment by andresmgot** [comment]
Thanks for reporting the issue @acim. We are aware of the issue, we are changing the syntax that define elements to hide in the JSON schema and we haven't updated the charts yet. Once we close https://github.com/kubeapps/kubeapps/pull/1720/ we will update all the charts in bulk with the new syntax. I will try to fix those as soon as possible.

**Comment by andresmgot** [comment]
Hi @acim just a note to let you know that we have released a new version that fixes the issue (v1.10.2). Take a look to the docs to know the new format: https://github.com/kubeapps/kubeapps/blob/master/docs/developer/basic-form-support.md#subsections

Sorry for the incoveniences!
